### ISSUE TITLE
Missing read receipts + clicking 'jump to last read receipt' on the memberinfo == a big error.

### ISSUE THREAD

**Issue by lampholder** [element-hq/element-web#6550]
From a rageshake.

**Comment by lampholder** [comment]
Related:
https://github.com/vector-im/riot-web/issues/5761
https://github.com/vector-im/riot-web/issues/6011

**Comment by thomas-profitt** [comment]
I had what sounds like this issue a minute ago:

I had a read receipt, then the other user in the 1:1 room read messages I had sent (including one that had since been redacted), and instead of showing a new read receipt, no read receipt was shown.

I hard-refreshed my browser (Firefox 61), and behavior didn't change. I checked on Riot Android (0.8.13 from F-Droid), and didn't see a read receipt in the room on it either.

I clicked "Jump to read receipt" in Riot-web, and the timeline kept scrolling up without stopping until I fiddled with the scrollwheel and it stopped.

The other user sent a message a moment afterward, and when I scrolled back down with the down arrow button, I saw a read receipt by their message.
